Integrations/Anomaliv2/Anomaliv2.py

```diff
@@ -0,0 +1,882 @@
+
+    if output:
+        report_outputs = {'ID': report_id, 'Status': status, 'Platform': platform, 'Verdict': verdict}
+        ec = {'ThreatStream.Sandbox(val.ID == obj.ID)': report_outputs}
+        return_outputs(tableToMarkdown(F"The analysis status for id {report_id}", report_outputs), ec, report_info)
+    return status, verdict
+
+
+def submit_report(submission_type, submission_value, submission_classification="private", report_platform="WINDOWS7",
+                  premium_sandbox="false", detail=None):
+    """
+        Detonates URL or file that was uploaded to war room to ThreatStream sandbox.
+    """
+    uploaded_file = None
+    files = None
+
+    data = {
+        'report_radio-classification': submission_classification,
+        'report_radio-platform': report_platform,
+        'use_premium_sandbox': premium_sandbox,
+    }
+    if detail:
+        data['detail'] = detail
+
+    if submission_type == 'file':
+        try:
+            # submission_value will be entry id of uploaded file to war room
+            file_info = demisto.getFilePath(submission_value)
+        except Exception:
+            return_error(F"Entry {submission_value} does not contain a file.")
+
+        uploaded_file = open(file_info['path'], 'rb')
+        files = {'report_radio-file': (file_info['name'], uploaded_file)}
+    else:
+        data['report_radio-url'] = submission_value
+
+    res = http_request("POST", "v1/submit/new/", params=CREDENTIALS, data=data, files=files)
+    # closing the opened file if exist
+    if uploaded_file:
+        uploaded_file.close()
+
+    if str(res.get('success', '')).lower() == 'true':
+        report_info = res['reports'][report_platform]
+        report_id = report_info['id']
+        report_status, _ = get_submission_status(report_id, False)
+
+        report_outputs = {'ID': report_id, 'Status': report_status, 'Platform': report_platform}
+        ec = {'ThreatStream.Sandbox': report_outputs}
+        return_outputs(tableToMarkdown(F"The submission info for {submission_value}", report_outputs), ec, report_info)
+    else:
+        demisto.results(F"The submission of {submission_value} failed")
+
+
+def get_report(report_id):
+    """
+        Returns the report from ThreatStream sandbox by id.
+    """
+    report = http_request("GET", F"v1/submit/{report_id}/report", params=CREDENTIALS, headers=HEADERS)
+    if not report:
+        demisto.results(F"No report found with id {report_id}")
+        sys.exit()
+    hm, ec = get_report_outputs(report.get('results', {}), report_id)
+
+    return_outputs(hm, ec, report)
+
+
+def add_tag_to_model(model_id, tags, model="intelligence"):
+    """
+        Adds tag to specific Threat Model. By default is set to intelligence (indicators).
+    """
+    data = {
+        'tags': [{'name': t.strip(), 'tlp': 'red'} for t in tags.split(',')]
+    }
+
+    res = http_request("POST", F"v1/{model}/{model_id}/tag/", params=CREDENTIALS, data=json.dumps(data))
```

Comment:
There is reason for json.dumps usage. In case of assigning directly the following error is received:
`Error in API call to ThreatStream 400 - {"message": "The format indicated 'application/x-www-form-urlencoded' had no available deserialization method. Please check your ``formats`` and ``content_types`` on your Serializer.", "success": false}`

Reply:
ok, got it